src/lib.rs

```diff
@@ -9,7 +9,7 @@
 //! [p256]: https://docs.rs/p256/
 
 #![no_std]
-#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg), feature(doc_auto_cfg))]
```

Comment:
:eyes:

Reply:
It does the condition documentation attribute automatically: https://github.com/rust-lang/rust/issues/43781